ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml

```diff
@@ -0,0 +1,108 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: openshift-pipelines-runner
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-serverless-rosa-classic-iib
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      AWS_REGION: us-east-1
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+              [
+                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Product to Product Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"], "jira_epic": "!default"}
+              ],
+          "success_rules": [{"jira_epic": "MSINTEROP-1162"}]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/rosa/p2p-msi.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.16-p2p","rosa-classic-p2p","pipelines-serverless-p2p"]'
+      FIREWATCH_DEFAULT_JIRA_COMPONENT: '["QA"]'
+      FIREWATCH_DEFAULT_JIRA_EPIC: MSINTEROP-1163
+      FIREWATCH_DEFAULT_JIRA_PROJECT: MSINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      INSTALL_FROM_IIB: "true"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.16"
+      OPERATOR1_CONFIG: name=openshift-pipelines-operator-rh;namespace=openshift-pipelines;channel=latest
+      OPERATOR2_CONFIG: name=serverless-operator;namespace=openshift-serverless;channel=stable
+      PIPELINES_TEST_SPECS: specs/clustertasks/clustertask-multiarch.spec
+      S3_BUCKET_OPERATORS_LATEST_IIB_PATH: ci-iib-jobs-trigger/operators-latest-iib.json
+    test:
+    - ref: operator-install
+    - ref: openshift-pipelines-deploy-tekton
+    - ref: openshift-pipelines-tests
+    - ref: p2p-additional-labels
+    workflow: firewatch-rosa-aws-sts
+- as: openshift-pipelines-serverless-rosa-classic-rc
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      CHANNEL_GROUP: candidate
```

Comment:
shouldn;t channel group for 4.17 be `nightly` here?

Reply:
I think we were just going to do nightly for development, then switch to candidate before merging. @rujutashinde can you confirm?

Reply:
yes that is correct nightly for development work and we switch to candidate when we are ready to run

Reply:
okay, got it, thanks for clarifying